packages/react-ui/src/widgets/TableWidgetUI/TableWidgetUI.js

```diff
@@ -89,10 +94,16 @@ function TableWidgetUI({
     onSetPage(0);
   };
 
+  const fixedHeightStyle = {};
+  if (height) {
+    const paginationHeight = paginationRef?.current?.clientHeight || 0;
+    fixedHeightStyle.height = `calc(${height} - ${paginationHeight}px)`;
+  }
+
   return (
     <>
-      <TableContainer>
-        <Table>
+      <TableContainer style={fixedHeightStyle}>
+        <Table stickyHeader size='small'>
```

Comment:
If this is gonna be 'hardcoded' then we should ping @Clebal about it for opinion

Reply:
We can easily expose another prop if needed.
But I really think this should be the default. This matches the designs of integration in builder and makes the table much more usable. The default cell padding of MUI is huge which is quite unpractical in most scenarios when space is constrained.

Reply:
I think Jose designed it using default size as medium.

Size small
![image](https://user-images.githubusercontent.com/9151432/149761422-941b2565-16cd-4763-8da5-01b8dd667305.png)

Size medium
![image](https://user-images.githubusercontent.com/9151432/149761472-e62ca1e0-e6f4-4b4c-8c82-e3a03205d249.png)

Jose's design
![image](https://user-images.githubusercontent.com/9151432/149761222-719232c4-8b9a-4d24-b67c-8dd6fcaf6e29.png)

In any case, **I think this should be a prop**.

Reply:
I like 'dense' approach @bbecquet 👏🏻

Reply:
@Clebal Alright :+1: 
As `size=small|medium` is MUI terminology and isn't very clear, I've expose a boolean prop `dense`, and set it to `false` by default. Builder will use `true`.
I've also added a case in TableWidgetUI storybook to illustrate that.